components/admin_console/cloud_migrate/widgets/progress_bar.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import * as React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import './progress_bar.scss';
+
+export interface ProgressBarProps {
+    percentage: number;
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = (props: ProgressBarProps) => {
+    const percetageText = `${props.percentage}%`;
+    return (
+        <div className='ProgressBar'>
+            <FormattedMessage
+                id='admin.general.importInProgress'
+                defaultMessage='Import in Progress'
+            />
```

Comment:
Nice to have would be to pass this in as a prop to also have it 100% agnostic to the import